fdbserver/DataDistribution.actor.cpp

```diff
@@ -1361,6 +1369,8 @@ ACTOR Future<Void> dataDistributor(DataDistributorInterface di, Reference<AsyncV
 	return Void();
 }
 
+namespace data_distribution_test {
```

Comment:
I haven't see this code before. What is the rationale behind encapsulating this?
The test that uses this seems like a generic flow test that has nothing to do with DD.
Does it make sense for it to be in this file?

Reply:
I'm thinking move those general tests to flow. Other added tests related to DD will be added in this namespace.